distribution/packages/src/deb/debmake_install.sh

```diff
@@ -90,5 +90,17 @@ for i in "${binary_files[@]}"; do
 	chmod -c 750 "$i"
 done
 
+# Append ".new" to configuration file names
+dot_new_config_files=()
+dot_new_config_files+=("${buildroot}/etc/init.d/${name}")
+dot_new_config_files+=("${buildroot}/${config_dir}/jvm.options")
+dot_new_config_files+=("${buildroot}/${config_dir}/opensearch.yml")
+dot_new_config_files+=("${buildroot}/${config_dir}/opensearch-security/roles.yml")
+dot_new_config_files+=("${buildroot}/${config_dir}/opensearch-security/internal_users.yml")
```

Comment:
This could be done in one go:

```bash
dot_new_config_files=(
    "${buildroot}/etc/init.d/${name}"
    "${buildroot}/${config_dir}/jvm.options"
    "${buildroot}/${config_dir}/jvm.options"
    "${buildroot}/${config_dir}/opensearch.yml"
    "${buildroot}/${config_dir}/opensearch-security/roles.yml"
    "${buildroot}/${config_dir}/opensearch-security/internal_users.yml"
)
```